### PR TITLE
Format headers of matching and numerics

### PR DESCRIPTION
**Pull request by angriman**
This is a continuation of #803. Changes do not create conflicts with existing PRs and are automatically generated with the exception of:
- move a few comments on a new line for better readability
- add missing `#include`; this was necessary because changing the order of some `#include`s broke the build

### CODE REVIEW

**Review by avdgrinten — CHANGES_REQUESTED** [review]
Non-automated parts of the PR should be in their own commits (for example, it's impossible to review which #include was added).

**Comment by angriman** [comment]
Yes good point, I moved those changes to separate commits. The missing include is fixed before the formatting to avoid commits where NetworKit would not build successfully.

File: include/networkit/algebraic/MatrixTools.hpp [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #include <atomic>
 #include <cmath>
 
-#include <networkit/algebraic/Globals.hpp>
+#include <networkit/algebraic/AlgebraicGlobals.hpp>
```
Comment (avdgrinten) [inline include/networkit/algebraic/MatrixTools.hpp:14]: Hm? Why is this changed? (Instead of doing this in the first commit?)

**Review by avdgrinten — COMMENTED** [review]
When you're fixing comments, you can also turn them into proper sentences with capitalization and punctuation. Other than that, this LGTM.

File: include/networkit/algebraic/MatrixTools.hpp (hunk 1 above)
Comment (angriman) [reply include/networkit/algebraic/MatrixTools.hpp:14]: Ah sorry, instead of using `git commit --fixup` to fix the first commit I used `git commit --amend`, I'll fix this now.

**Review by fabratu — APPROVED** [review]
LGTM.

**Comment by angriman** [comment]
Thanks for the review. I need another one from @avdgrinten due to the requested changes.

**Review by avdgrinten — APPROVED** [review]
